tmt-app/src/Components/Quiz.jsx

```diff
@@ -29,6 +28,27 @@ function Quiz() {
           </Grid>
         ))}
       </Grid>
+      <Link
+        sx={{ color: 'Boba.main' }}
+        underline="none"
+        aria-label="Learn more about other different majors"
+        component={RouterLink}
+        to="/quiz/skills"
+      >
+        <PrimaryBtn
+          sx={{
+            backgroundColor: 'Matcha.main',
+            '&.MuiButtonBase-root': {
+              padding: {
+                xs: '1rem 2.5rem',
+                xl: '1rem 3.25rem',
+              },
+            },
+          }}
+        >
+          <Typography variant="h4">Next</Typography>
```

Comment:
I don't think this should be an h4 variant, button text isn't normally a heading. And as a habit we shouldn't get used to treating and using heading elements just for sizing

Reply:
Our custom h4 variant matches the style we need for this button. Should I create another variant just for buttons?

Reply:
I dont think we need another variant we can just leave the text here, and let the button style it. Buttons can apply text styling as well. and we can consider the primary/secondary button variants per the design spec